ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -797,7 +797,7 @@ def test_swapaxes(
 @given(
     x_min_n_max=helpers.dtype_and_values(
         available_dtypes=ivy_np.valid_numeric_dtypes, num_arrays=3
-    ),
+    ).filter(lambda x_min_n_max: x_min_n_max[0][0] == x_min_n_max[0][1] and x_min_n_max[0][1] == x_min_n_max[0][2]),
```

Comment:
does this logic work correctly? you are performing the `==` operator on two lists, do we only care about checking for total equality between the lists? I would have thought elementwise equality is more important

Reply:
Actually here x_min_n_max is an object of class ivy.FloatDtype, and I want to ensure all the types are the same since I observed some overflow when dtypes are different.